firefox-desktop/views/mobile_usage_2021.view.lkml

```diff
@@ -0,0 +1,81 @@
+view: mobile_usage_2021 {
+  sql_table_name: `mozdata.telemetry.mobile_usage_2021`
+    ;;
+
+  dimension: app_name {
+    type: string
+    sql: ${TABLE}.app_name ;;
+  }
+
+  dimension: campaign {
+    type: string
+    sql: ${TABLE}.campaign ;;
+  }
+
+  dimension: canonical_app_name {
+    type: string
+    sql: ${TABLE}.canonical_app_name ;;
+  }
+
+  measure: cdou {
+    type: number
+    sql: ${TABLE}.cdou ;;
+  }
+
+  dimension: channel {
+    type: string
+    sql: ${TABLE}.channel ;;
+  }
+
+  dimension: country {
+    type: string
+    map_layer_name: countries
+    sql: ${TABLE}.country ;;
+  }
+
+  measure: dau {
+    type: number
+    sql: ${TABLE}.dau ;;
+  }
```

Comment:
Did you work on the mobile KPI stuff yet? Just checking because this looks like the default import and it doesn't include anything about the joined forecasts. This should probably be a sum measure.